templates/konvoy-ui.yaml

```diff
@@ -0,0 +1,234 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1alpha1
+kind: Addon
+metadata:
+  name: opsportal
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: opsportal
+  annotations:
+    appversion.kubeaddons.mesosphere.io/opsportal: "1.0.0"
+spec:
+  kubernetes:
+    minSupportedVersion: v1.15.0
+  manifest: |
```

Comment:
Manifest based addons are no longer supported @s12chung, do we need to convert this to a helm chart?